packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -372,6 +380,10 @@ export class PeoplePickerTypesExample extends BaseComponent<any, IPeoplePickerEx
     return this._filterPromise(mostRecentlyUsed);
   }
 
+  private _setRef(component: IBasePicker<IPersonaProps>): void {
```

Comment:
BaseComponent has a method called _resolveRef which you can use rather than setRef.